src/redux/services/resourceRefs.ts

```diff
@@ -694,3 +701,28 @@ export function findResourcesToReprocess(resource: K8sResource, resourceMap: Res
 
   return resources;
 }
+
+/**
+ * Reprocess all optoinal refs for the specified resources
+ */
+
+export function reprocessOptionalRefs(
```

Comment:
Same as before it's better to have an object and send properties there than sending multiple arguments to a function